game/domain/map.lua

```diff
@@ -14,7 +14,9 @@ function Map:init(sector_name)
 
   ELEMENT.init(self)
 
-  local specs = DB.loadDomain(sector_name)
+  local specs = SECTORS[sector_name]
```

Comment:
Alrighty.
My reasoning for this was that well, if we are using only the sector-section of the DB _and_ it's only one file, we might as well save that entry itself in a "const" in the Sector class.

I suppose calling`DB` explicitly makes it clearer to other people what is happening.